modules/components/org.wso2.carbon.event.simulator.core/src/main/java/org/wso2/eventsimulator/core/EventSimulatorStart.java

```diff
@@ -0,0 +1,252 @@
+package org.wso2.eventsimulator.core;
+
+import org.json.JSONObject;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.wso2.eventsimulator.core.simulator.bean.FeedSimulationDto;
+import org.wso2.eventsimulator.core.simulator.bean.FeedSimulationStreamConfiguration;
+import org.wso2.eventsimulator.core.simulator.exception.EventSimulationException;
+import org.wso2.eventsimulator.core.simulator.singleventsimulator.SingleEventDto;
+import org.wso2.eventsimulator.core.util.EventSimulatorParser;
+import org.wso2.eventsimulator.core.util.EventSimulatorPoolExecutor;
+import scala.util.parsing.json.JSON;
+
+import java.util.ArrayList;
+import java.util.Map;
+import java.util.UUID;
+import java.util.concurrent.ConcurrentHashMap;
+
+/**
+ * Created by ruwini on 2/18/17.
+ */
```

Comment:
Remove above comment and add proper class comments.. Check other places as well.